relayer/query.go

```diff
@@ -130,6 +130,50 @@ func (c *Chain) QueryClientState(height int64) (*clienttypes.QueryClientStateRes
 	return clientutils.QueryClientStateABCI(c.CLIContext(height), c.PathEnd.ClientID)
 }
 
+// QueryUnpackedClientState retrevies the latest consensus state for a client in state at a given height
+// and unpacks it to exported client state interface
+func (c *Chain) QueryUnpackedClientState(height int64) (ibcexported.ClientState, error) {
```

Comment:
I'd name this `QueryClientState` and rename the old function to `QueryClientStateResponse`